### PR TITLE
Modified TCP route to include SNI hostname field. (#1)

### PR DESCRIPTION
**Pull request by pnikonowicz**

### CODE REVIEW

**Comment by selzoc** [comment]
@pivotal-jbarrett can you please make sure to sign the CLA and update this PR as well?

**Comment by pnikonowicz** [comment]
we reset the author to pass the CLA checks

**Comment by selzoc** [comment]
Hi @xtreme-prasath , can you please sign the CLA?

**Review by Benjamintf1 — COMMENTED** [review]
did you mean to change the readme too?

File: README.md [hunk 1]
```diff
@@ -92,6 +93,20 @@ Please report all issues and feature requests in [cloudfoundry/routing-release](
...
 
+### SNI Routing
+The route registrar can be used to setup SNI routing. This is an example route json:
+```
+{
+  "routes": [
+    {
+      "external_port": "TLS_PORT_OF_ROUTE_SOURCE",
+      "name": "SOME_ROUTE_NAME",
+      "port": "TLS_PORT_OF_ROUTE_DESTINATION",
```
[the remark this answers is not in the document]
Comment (xtreme-prasath) [reply README.md:104]: Great catch, we missed updating this bit.

**Pull request merged**